bluesky/plan_tools.py

```diff
@@ -0,0 +1,82 @@
+import matplotlib.pyplot as plt
+from matplotlib import collections as mcollections
+from matplotlib import patches as mpatches
+
+
+def path_plotter(plan, x_motor, y_motor, ax=None, probe_size=None):
```

Comment:
Can I bikeshed this name to `plot_path`? This sounds like a returns a factory. I'm perfectly happy with the module name `plan_tools` though.